From 849584709f2c14d978cdf37af7e146e59b49d994 Mon Sep 17 00:00:00 2001
From: Salman Wahib <sxlmnwb.dev@gmail.com>
Date: Tue, 26 May 2026 16:13:47 +0700
Subject: [PATCH 06/18] =?UTF-8?q?time:=20reduce=20default=20timer=5Fslack?=
 =?UTF-8?q?=5Fns=20from=2050=C2=B5s=20to=2050ns?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

tighter hrtimer coalescing window for nanosleep/select/futex
inherited by all processes from PID 1

Co-authored-by: Mauri de Souza Meneguzzo <mauri870@gmail.com>
Signed-off-by: Salman Wahib <sxlmnwb.dev@gmail.com>
---
 init/init_task.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/init/init_task.c b/init/init_task.c
index 5c838757fc10..34272eaaa660 100644
--- a/init/init_task.c
+++ b/init/init_task.c
@@ -169,7 +169,7 @@ struct task_struct init_task __aligned(L1_CACHE_BYTES) = {
 	.journal_info	= NULL,
 	INIT_CPU_TIMERS(init_task)
 	.pi_lock	= __RAW_SPIN_LOCK_UNLOCKED(init_task.pi_lock),
-	.timer_slack_ns = 50000, /* 50 usec default slack */
+	.timer_slack_ns = 50, /* 50 nsec default slack */
 	.thread_pid	= &init_struct_pid,
 	.thread_node	= LIST_HEAD_INIT(init_signals.thread_head),
 #ifdef CONFIG_AUDIT
-- 
2.54.0

